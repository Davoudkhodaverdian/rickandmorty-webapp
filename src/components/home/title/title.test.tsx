import '@testing-library/jest-dom'
import renderer from 'react-test-renderer';
import Title from '.';

test('test home page', async () => {
  // ARRANGE
  const component = renderer.create(
    <Title />,
  );
  let tree = component.toJSON()!;
  expect(tree).toMatchSnapshot();

  // re-rendering
  (tree as any) = component.toJSON();
  expect(tree).toMatchSnapshot();
})


