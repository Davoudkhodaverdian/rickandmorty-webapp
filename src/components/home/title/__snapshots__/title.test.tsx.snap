// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test home page 1`] = `
<div>
  home page
</div>
`;

exports[`test home page 2`] = `
<div>
  home page
</div>
`;
